packages/remix-react/transition.ts

```diff
@@ -790,7 +790,7 @@ export function createTransitionManager(init: TransitionManagerInit) {
     key: string,
     match: ClientMatch
   ) {
-    if (typeof AbortController === "undefined") {
+    if (typeof window === "undefined") {
```

Comment:
Please use document instead of window. Deno is a runtime we are trying to support and it has window defined in the server but not document.